src/main/java/com/redhat/devtools/intellij/qute/psi/internal/QuteJavaConstants.java

```diff
@@ -1,87 +1,88 @@
 /*******************************************************************************
-* Copyright (c) 2021 Red Hat Inc. and others.
-* All rights reserved. This program and the accompanying materials
-* which accompanies this distribution, and is available at
-* http://www.eclipse.org/legal/epl-v20.html
-*
-* SPDX-License-Identifier: EPL-2.0
-*
-* Contributors:
-*     Red Hat Inc. - initial API and implementation
-*******************************************************************************/
+ * Copyright (c) 2021 Red Hat Inc. and others.
+ * All rights reserved. This program and the accompanying materials
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Red Hat Inc. - initial API and implementation
+ *******************************************************************************/
 package com.redhat.devtools.intellij.qute.psi.internal;
 
 /**
  * Qute Java constants.
- * 
- * @author Angelo ZERR
  *
+ * @author Angelo ZERR
  */
 public class QuteJavaConstants {
 
-	public static final String JAVAX_INJECT_NAMED_ANNOTATION = "javax.inject.Named";
+    public static final String JAVA_LANG_OBJECT_TYPE = "java.lang.Object";
+
+    public static final String JAVAX_INJECT_NAMED_ANNOTATION = "javax.inject.Named";
```

Comment:
Indeed but I will do that in an another PR, because this support is done in an another commit. I would like to follow each commits from quarkus-ls please.